File: dotnet-maui/ReservationsDotnetMaui/Models/ConferenceRoom.cs
namespace ReservationsDotnetMaui.Models;

public class ConferenceRoom
{
    public string Id { get; set; } = string.Empty;
    public string Name { get; set; } = string.Empty;
    public string OfficeId { get; set; } = string.Empty;
    public int Capacity { get; set; }
    public bool IsReserved { get; set; }
    public string? ReservedBy { get; set; }
    public DateTime? ReservedUntil { get; set; }
}

File: flutter/flutterapp/lib/models/conference_room.dart
class ConferenceRoom {
  ConferenceRoom({
    required this.id,
    required this.name,
    required this.officeId,
    required this.capacity,
    this.isReserved = false,
    this.reservedBy,
    this.reservedUntil,
  });

  final String id;
  final String name;
  final String officeId;
  final int capacity;
  bool isReserved;
  String? reservedBy;
  DateTime? reservedUntil;
}
